cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,21 +491,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
```

Comment:
It is my understanding that in newer versions of JQuery, native focus events are used. That's when the issue cropped up.